src/main/java/uk/gov/register/views/ViewFactory.java

```diff
@@ -24,24 +26,24 @@
     private final ItemConverter itemConverter;
     private final PublicBodiesConfiguration publicBodiesConfiguration;
     private final GovukOrganisationClient organisationClient;
-    private final String registerDomain;
     private final RegisterData registerData;
     private final RegisterDomainConfiguration registerDomainConfiguration;
+    private final RegisterContentPages registerContentPages;
 
     @Inject
     public ViewFactory(RequestContext requestContext,
                        ItemConverter itemConverter,
                        PublicBodiesConfiguration publicBodiesConfiguration,
                        GovukOrganisationClient organisationClient,
-                       RegisterDomainConfiguration domainConfiguration,
+                       RegisterConfiguration registerConfiguration,
```

Comment:
Rather than injecting the whole `RegisterConfiguration` class, I think it's better to inject a minimal interface (in this case, we might want to create a `RegisterContentPagesConfiguration` interface?).  These smaller interfaces are much easier to stub out in tests and it makes it clearer what the class is actually depending on.